tools/test.py

```diff
@@ -83,6 +86,40 @@ def main():
     if args.show or args.show_dir:
         cfg = trigger_visualization_hook(cfg, args)
 
+    if args.tta:
+        if 'tta_model' not in cfg:
+            cfg.tta_model = dict(
+                type='DetTTAModel',
+                tta_cfg=dict(
+                    nms=dict(type='nms', iou_threshold=0.5), max_per_img=100))
+        if 'tta_pipeline' not in cfg:
+            if cfg.test_dataloader.dataset.type in ('MultiImageMixDataset',
```

Comment:
This way of writing is not general enough. You can refer to the previous similar writing. https://github.com/open-mmlab/mmdetection/blob/master/tools/analysis_tools/optimize_anchors.py#L345